package/MDAnalysis/analysis/encore/similarity.py

```diff
@@ -861,10 +861,10 @@ def hes(ensembles,
     # on the universe.
     if align:
         for ensemble in ensembles:
-            mda.analysis.align.rms_fit_trj(ensemble, ensembles[0],
+            mda.analysis.align.AlignTraj(ensemble, ensembles[0],
                                            select=selection,
```

Comment:
Minor remark. The indentation is now off by 2 spaces here. If you could change it here and for the other parameters as well that would be nice.